spec/lib/data_magic/search_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
 require 'data_magic'
 require 'fixtures/data.rb'
+require 'pry'
```

Comment:
Hmm, yes ... I thought it's okay to leave it in the spec files in case I got back to these specs later but I can remove it,